src/core/shaders/graphs/metallicroughness.frag.json

```diff
@@ -1924,17 +1926,6 @@
                     ],
                     "substitution": "vec4 $outputColor = kuesa_metalRoughFunction($baseColor, $metalness, $roughness, $ambientOcclusion, $worldPosition, $worldView, $worldNormal);"
                 },
-                {
-                    "format": {
```

Comment:
ditto

Reply:
I have load the prototypes in the kuesa folder, open the .qt3d file and save/export. Maybe the prototypes in the repo are incorrect?

Reply:
done (by updating the prototypes both in the repo and the nodeeditor)